examples/apollo-watch-fragments/package.json

```diff
@@ -12,7 +12,7 @@
     "react-dom": "^17.0.2"
   },
   "scripts": {
-    "start": "yarn generate-resolver-typings && concurrently 'yarn duct-tape-compiler --watch' 'yarn serve'",
+    "start": "yarn generate-resolver-typings && yarn concurrently \"yarn duct-tape-compiler --watch\" \"yarn serve\"",
```

Comment:
This is for Windows support, right?

Reply:
I changed it slightly now to just escape " character and tested on windows bash, powershell and WSL. Hoep it's fine :)